docs/PORTABLE_SERVICES.md

```diff
@@ -243,6 +250,34 @@ image. To facility 3 and 4 you also need to include a boot loader in the
 image. As mentioned `mkosi -b` takes care of all of that for you, but any other
 image generator should work too.
 
+## Layered Images
```

Comment:
I don't like the name "layered" btw, it suggests obstruction of files in lower layers by upper layers. I'd always talk about "extension images", which I think is clearer, as it underlines that the contained files "extend" (and not obstruct/override) the underlying image. I mean, ultimately people can do with the images what they want, but i'd pick a nomenclature here that pushes people towards extending, not overriding.

Reply:
the naming "extension images" also has the benefit that it makes clear that there's one main thing, and then extension things for it, i.e. that the objects involved are not equally footed, but one is more important and the others are just secondary concepts to that. the word "layer" does not transport this distinction.

Reply:
I'll change the title and the description, but I'll keep one mention of "layering" as that's a familiar term in the container world. What we do here in practice is the runtime-equivalent of building a podman image from multiple layers - with the key differentiator that we combine at runtime, rather than buildtime